### PR TITLE
Update phrasing on `ultrasonic` sensor/camera to recommend specific hardware

### PR DESCRIPTION
**Pull request by sguequierre**
Re. conversation with Rand-- we haven't actually tested anything else so just recommending this model is best for now. Rand has signed off on this and un-requires her review

### CODE REVIEW

File: docs/components/camera/_index.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ For configuration information, click on the model name:
 | [`align_color_depth_homography`](align-color-depth-homography/) | Uses a homography matrix to align the color and depth images. |
 | [`join_pointclouds`](join-pointclouds/) | Combines the point clouds from multiple camera sources and projects them to be from the point of view of target_frame. |
 | [`transform`](transform/) | A pipeline for applying transformations to an input image source. |
-| [`ultrasonic`](ultrasonic/) | An ultrasonic distance sensor like the [HC-S204](https://www.sparkfun.com/products/15569). |
+| [`ultrasonic`](ultrasonic/) | An [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```
Comment (sguequierre) [inline docs/components/camera/_index.md:60]:
```suggestion
| [`ultrasonic`](ultrasonic/) | [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```

File: docs/components/camera/_index.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ For configuration information, click on the model name:
 | [`align_color_depth_homography`](align-color-depth-homography/) | Uses a homography matrix to align the color and depth images. |
 | [`join_pointclouds`](join-pointclouds/) | Combines the point clouds from multiple camera sources and projects them to be from the point of view of target_frame. |
 | [`transform`](transform/) | A pipeline for applying transformations to an input image source. |
-| [`ultrasonic`](ultrasonic/) | An ultrasonic distance sensor like the [HC-S204](https://www.sparkfun.com/products/15569). |
+| [`ultrasonic`](ultrasonic/) | [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```
Comment (andf-viam) [inline docs/components/camera/_index.md:60]:
```suggestion
| [`ultrasonic`](ultrasonic/) | The [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```

Or `Supports the ...` or similar. Just to make this also a sentence, matching above.

**Review by andf-viam — APPROVED** [review]
LGTM, one nit comment for the tables. Thank you!

File: docs/components/sensor/_index.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ For configuration information, click on the model name:
 | Model | Description |
 | ----- | ----------- |
 | [`fake`](fake/) | A model used for testing, with no physical hardware. |
-| [`ultrasonic`](ultrasonic/) | An ultrasonic distance sensor like the [HC-S204](https://www.sparkfun.com/products/15569) |
+| [`ultrasonic`](ultrasonic/) | [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor |
```
Comment (andf-viam) [inline docs/components/sensor/_index.md:46]: One more lil spot!

```suggestion
| [`ultrasonic`](ultrasonic/) | The [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor |
```

**Pull request merged**